src/Redist/Microsoft.ML.DnnImageFeaturizer.ModelRedist/Microsoft.ML.DnnImageFeaturizer.ModelRedist.proj

```diff
@@ -11,14 +11,14 @@
 
   <ItemGroup>
     <ModelFiles Include="$(ModelPlacementDir)\AlexNetPrepOnnx\AlexNetPreprocess.onnx"
-    Url="https://aka.ms/mlnet-resources/image/AlexNetPrepOnnx/AlexNetPreprocess.onnx "
+    Url="https://aka.ms/mlnet-resources/image/AlexNetPrepOnnx/AlexNetPreprocess.onnx"
     DestinationFile="$(ModelPlacementDir)\AlexNetPrepOnnx\AlexNetPreprocess.onnx"
```

Comment:
Was the space causing a problem in Linux only?

Reply:
I haven't try on mac but this cause download failure on Linux not on Windows

---
In reply to: [501872554](https://github.com/dotnet/machinelearning/pull/5423#discussion_r501872554) [](ancestors = 501872554)